readme.md

```diff
@@ -1 +1,192 @@
 # Nanobots data upgrader
+
+Nanobots is a library which helps you fix data and refactor your database in a safe, reliable way!
+
+## Problems with breaking schema changes
+
+There are a few problems with making a breaking schema change (like a rename), and many teams/companies just choose not to make them. This means your database cannot be refactored and evolved like you code, which can cause problems downt the track.
```

Comment:
Bit of grammar weirdness in here but the one that you might want to change is `...like you code` and `downt`

Reply:
Making a few small changes